pkg/vadmin/create_db_vc.go

```diff
@@ -47,6 +49,21 @@ func (v *VClusterOps) CreateDB(ctx context.Context, opts ...createdb.Option) (ct
 	vopts := v.genCreateDBOptions(&s, certs)
 	vdb, err := v.VCreateDatabase(&vopts)
 	if err != nil {
+		// If it is determined that vertica is already running, we may allow the
+		// failure to continue. This could be a result of a previous failed
+		// create db that was able to partially complete but ultimately failed
+		// before updating the status condition to indicate that the create db
+		// was finished. By ignoring the error this time, we will be able to
+		// update the status condition and start any nodes that may be down in
+		// order to bring the cluster online.
+		dbIsRunningError := &vops.DBIsRunningError{}
```

Comment:
What do you mean by partially complete? Do you mean on the operator side?

Reply:
No, this is the server side. The cases we saw is that create db fails during the package installation. If we got far enough to bring up a vertica process, chances are the database is fine to use.